src/store.js

```diff
@@ -252,6 +252,8 @@ function resetStoreVM (store, state, hot) {
 
   // bind store public getters
   store.getters = {}
+  // reset local getters cache
+  makeLocalGettersCache = {}
```

Comment:
Let's initialize it with `Object.create(null)` to avoid false positive check with its prototype functions.